tests/release/pipelines/fbc_release.go

```diff
@@ -173,6 +180,58 @@ var _ = framework.ReleasePipelinesSuiteDescribe("FBC e2e-tests", Label("release-
 			})
 		})
 	})
+
+	Describe("with FBC pre-GA process", Label("fbcPreGA"), func() {
+		var component *appservice.Component
+
+		BeforeAll(func() {
+			devFw, err = framework.NewFrameworkWithTimeout(
+				devWorkspace,
+				time.Minute*60,
+				stageOptions,
+			)
+			Expect(err).NotTo(HaveOccurred())
+
+			managedFw, err = framework.NewFrameworkWithTimeout(
+				managedWorkspace,
+				time.Minute*60,
+				stageOptions,
+			)
+			Expect(err).NotTo(HaveOccurred())
+
+			_, err = devFw.AsKubeDeveloper.HasController.CreateApplication(fbcPreGAAppName, devNamespace)
+			Expect(err).NotTo(HaveOccurred())
+
+			_, err = devFw.AsKubeDeveloper.ReleaseController.CreateReleasePlan(fbcPreGARPName, devNamespace, fbcPreGAAppName, managedNamespace, "true")
+			Expect(err).NotTo(HaveOccurred())
+
+			createFBCReleasePlanAdmission(fbcPreGARPAName, *managedFw, devNamespace, managedNamespace, fbcPreGAAppName, fbcPreGAECPolicyName, relSvcCatalogPathInRepo, "true", issueId, productName, productVersion)
+			component = releasecommon.CreateComponentByCDQ(*devFw, devNamespace, managedNamespace, fbcPreGAAppName, fbcPreGACompName, fbcSourceGitURL)
+			createFBCEnterpriseContractPolicy(fbcPreGAECPolicyName, *managedFw, devNamespace, managedNamespace)
+		})
+
+		AfterAll(func() {
+			if !CurrentSpecReport().Failed() {
+				Expect(devFw.AsKubeDeveloper.HasController.DeleteApplication(fbcPreGAAppName, devNamespace, false)).NotTo(HaveOccurred())
+				Expect(managedFw.AsKubeDeveloper.TektonController.DeleteEnterpriseContractPolicy(fbcPreGAECPolicyName, managedNamespace, false)).NotTo(HaveOccurred())
+				Expect(managedFw.AsKubeDeveloper.ReleaseController.DeleteReleasePlanAdmission(fbcPreGARPAName, managedNamespace, false)).NotTo(HaveOccurred())
+			}
+		})
+
+		var _ = Describe("FBC pre-GA post-release verification", func() {
+			It("verifies that a build PipelineRun is created in dev namespace and succeeds", func() {
+				assertBuildPipelineRunCreated(*devFw, devNamespace, managedNamespace, fbcPreGAAppName, component)
+			})
+
+			It("verifies the fbc release pipelinerun is running and succeeds", func() {
```

Comment:
@theflockers are there any pipeline results or anything that we should check are populated in the pre-GA case? Or anything at all to check here for a little sanity?

Reply:
Not really. We have a few results but the two relevant ones only saves the information about the image built by IIB. We only care about the result of the task itself, which later in the pipeline is used to publish. But this does not happen here (publishing) because a pyxis flag is required to trigger publishing task.